pubsub/subscriptions/subscription_test.go

```diff
@@ -773,3 +810,43 @@ func getOrCreateSub(ctx context.Context, client *pubsub.Client, subID string, cf
 	}
 	return sub, nil
 }
+
+func createBigQueryTable(projectID, datasetID, tableID string) error {
+	ctx := context.Background()
+
+	c, err := bigquery.NewClient(ctx, projectID)
+	if err != nil {
+		return fmt.Errorf("error instantiating bigquery client: %v", err)
+	}
+	dataset := c.Dataset(datasetID)
+	if err = dataset.Create(ctx, &bigquery.DatasetMetadata{Location: "US"}); err != nil {
+		return fmt.Errorf("error creating dataset: %v", err)
+	}
+
+	table := dataset.Table(tableID)
+	schema := []*bigquery.FieldSchema{
+		{Name: "data", Type: bigquery.BytesFieldType, Required: true},
```

Comment:
In other PRs, this was a `StringFieldType` but that was giving me a type mismatch error.